addon/components/code-block.js

```diff
@@ -1,27 +1,11 @@
-/* global Prism */
-import Ember from 'ember';
-const {
-  Component,
-  computed
-} = Ember;
+import CodeBase from './code-base';
 
-export default Component.extend({
-  tagName:           'pre',
-  classNames:        ['code-block'],
-  classNameBindings: ['languageClass'],
+export default CodeBase.extend({
+  tagName: 'pre',
+  classNames: ['code-block'],
+  inline: false,
```

Comment:
you could make `true`, or `false` be a default in codebase

Reply:
Good point